pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -227,3 +227,81 @@ func (i *Inode) Open(ctx context.Context, rp *vfs.ResolvingPath, vfsd *vfs.Dentr
 	}
 	return fd.VFSFileDescription(), nil
 }
+
+// Stat implements kernfs.Inode.Stat.
+func (i *Inode) Stat(ctx context.Context, fs *vfs.Filesystem, opts vfs.StatOptions) (linux.Statx, error) {
+	conn := fs.Impl().(*filesystem).conn
+	task, creds := kernel.TaskFromContext(ctx), auth.CredentialsFromContext(ctx)
+	if task == nil {
+		panic("Stat must be called from a task context")
+	}
+
+	var in linux.FUSEGetAttrIn
+	// We don't set any attribute in the request, becuase in VFS2 fstat(2) will
+	// finally be translated into vfs.FilesystemImpl.StatAt() (see
+	// pkg/sentry/syscalls/linux/vfs2/stat.go), resulting in the same flow
+	// as stat(2). Thus GetAttrFlags and Fh variable are currently not used.
```

Comment:
Add a TODO to support `fstat(2)`.

Reply:
In this comment I mean `fstat(2)` will eventually fall into the same logic in VFS2 so it is covered (and don't need a TODO). If we're using VFS2, then there isn't a syscall we will set `GetAttrFlags` and `Fh`. So we are always sending the FUSE request w/o these two variable set. Slightly changed the comment to make it more clear.